skills/inferskills.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+import json
+
+
+class InferSkills:
+  def __init__(self):
+    self.vals = {}
+    with open('skillsets_labels.tsv', 'r') as f:
```

Comment:
These are only the top-n skills related to a particular skill-set. We need to remember to expand this to the list of all skills with more than k occurrences.